example/App.tsx

```diff
@@ -1,10 +1,22 @@
 import { StatusBar } from 'expo-status-bar';
 import React from 'react';
 import { StyleSheet, View } from 'react-native';
+import { useEffect } from 'react';
 
 import * as Maps from 'expo-maps';
+import * as Location from 'expo-location';
 
 export default function App() {
+
+  const getLocationPermissions = async () => {
+    await Location.requestForegroundPermissionsAsync();
+  }
+
+  useEffect(() => {
+    getLocationPermissions();
+  }, []);
```

Comment:
I'm wondering if user should do this in his App before using `Expo Maps ` or maybe if we should do this as a part of library functionalities. What do you think?

Reply:
This is only a temporary solution. I believe finally it should be a part of library functionalities.

In my opinion we should create separate issue to work on this topic. The solution will also depend strongly whether we'd like to keep using expo-location or switch to native ways of getting location permissions from user.

Reply:
Sound reasonable 👍